apps/docs/stories/theme.tsx

```diff
@@ -0,0 +1,6 @@
+const theme = `
```

Comment:
Leaving this empty for now, until we do more work on the storybook examples for Checkout